modules/backendscheduler/config.go

```diff
@@ -42,5 +43,14 @@ func ValidateConfig(cfg *Config) error {
 		return err
 	}
 
+	// Validate the measurement interval is twice the speed of the prune interval
+	// so that the when newBlockSelector is called it has enough time to delete
+	// the temporary entry and know that it has been persisted to the work cache.
+	// If the prune age is too short, work could get deleted before the
+	// newBlockSelector is able to delete the temporary entry.
+	if cfg.ProviderConfig.Compaction.MeasureInterval > cfg.Work.PruneAge/2 {
```

Comment:
Is the error message backward?  This is checking that `prune_age` is at least twice the `measurement interval`.

Reply:
I'll push a commit, but let me know if that still doesn't ring to you and we'll update.